### PR TITLE
Added capability for the dynamicHeight attribute to work without images

### PR DESCRIPTION
**Pull request by rokit**
Created this for issue #210 that I raised.

### CODE REVIEW

**Comment by rokit** [comment]
If there is an image present, it will work as before, otherwise it will just return the height of the first div in the item.

File: src/components/Carousel.js [hunk 1]
```diff
@@ -430,9 +430,11 @@ class Carousel extends Component {
         return images && images[selectedItem];
     }
 
-    getVariableImageHeight = (position) => {
-        const item = this.refs[`item${position}`];
-        const images = item && item.getElementsByTagName('img');
+    getVariableHeight = (position) => {
+				const item = this.refs[`item${position}`];
+				const images = item && item.getElementsByTagName('img');
+				const divs = item && item.getElementsByTagName('div');
```
Comment (leandrowd) [inline src/components/Carousel.js]: There is no requirement saying that each slide should be a div. We just iterate over children and then build a list of slides. This change wouldn't be robust enough.

Comment (driskell) [reply src/components/Carousel.js]: Would just using clientHeight of the <li> work? I guess might need to adjust CSS to ensure no padding but would account for entire slide no matter the content as long as nothing out-of-flow

Comment (leandrowd) [reply src/components/Carousel.js]: Sounds like a good idea. Need to be tested, though.

**Comment by leandrowd** [comment]
Hey @rokit, will you continue with this PR? Thanks

**Comment by sezna** [comment]
Hey @rokit, I’m closing this due to inactivity. Feel free to submit another PR if you work on this again.

**Pull request closed without merging**